### ISSUE TITLE
I can't get my head around the technical integration of oRTB

### ISSUE THREAD

**Issue by timt56** [openrtb/OpenRTB#31]
Hey, I'm only a beginner at this, so you'll have to allow some room for mistakes with me, sorry in advance!
So far, I've seen a half-dozen example oRTB exchanges, and read quite a bit of the documentation by third parties on GitHub.
However, when I look at the IAB docs and review the aforementioned projects for how they actually communicate, my conclusion is that oRTB, on the SSP side, is essentially just receiving JSON files via API requests, holding the auction (I couldn't find any documentation on a standardised way of doing this - can I do it any way I like?) and then issuing a response to the bidders and showing the creative. 
My question is, as I'm not well-versed in Java, can't I just run this system - receiving and sending files - on a different language?
Thanks, Tim

**Comment by manigandham** [comment]
Hey @timt56 

Yes, real-time bidding (RTB) is just servers from SSPs, DSPs, and all the vendors in the middle communicating over HTTP with JSON payloads. There are some exchanges (like Google) that also use Protobuf.

Every SSP has it's own process for holding an auction - how you evaluate an bid request and put a value on it is up to you.

And yes, you can use whatever language/framework/system you like to do this. This is a very old repo and not actually used by anyone I know. Most companies build their own stack using languages, infrastructure and tooling of their choice.

**Comment by timt56** [comment]
Cool. Is there anything I need to know about standardized URL schemes? As in, where to send bid requests, etc., or is this information different from DSP to DSP?

**Comment by manigandham** [comment]
No, every exchange will have their own URLs and schemas to follow.

**Comment by timt56** [comment]
Cool. Then I have one more question (sorry): is it feasible to build the auction system using PHP, or is the language unsuitable?

**Comment by manigandham** [comment]
You can use whatever language you want, the pros/cons are up to you.

RTB is performance intensive so usually implemented in Java, C#, C++, etc.
